assets/src/edit-story/components/helpCenter/companion/index.js

```diff
@@ -39,12 +44,18 @@ export function Companion({ tipKey, onTipSelect, isLeftToRightTransition }) {
           {...tip}
         />
       ) : (
-        <Menu key={'menu'} transitionKey={'menu'} onTipSelect={onTipSelect} />
+        <Menu
+          read={read}
+          key={'menu'}
+          transitionKey={'menu'}
```

Comment:
I know this wasn't you but Nit: we don't need string interpolation here
```suggestion
          key="menu"
          transitionKey="menu"
```

Reply:
Yea we can also move `transitionKey` out of here. removed them from the components but forgot to update this file